qcengine/programs/gcp.py

```diff
@@ -160,16 +160,25 @@ def build_input(
             # thus code blocks with FILE below are not used yet.
             # 'file',
         ]
+        # some methods not available in legacy version
+        mctc_gcp_levels = ["B973C", "R2SCAN3C"]
```

Comment:
B97-3c should be supported with the original version of gcp already.

Reply:
Holger was pretty sure otherwise. See "Dev notes" at https://github.com/psi4/psi4/pull/2842#issue-1518829460 . v2.0.2 at https://anaconda.org/psi4/gcp/files is the relevant "classic" one that I'm trying to wean psi4 folks off of. mctc-gcp is preferred as of psi4 v1.8.